### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by soroshsabz**
ITNOA

Hi,

I add CONTRIBUTING.md for this project based on Apache Thrift contributing file.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+If you already have a pull request outstanding, you will need to do a "force push" to overwrite it since you changed your commit history:
+
+    git push -u origin feature/9999-make-perfect --force
+
+A more detailed walkthrough of a squash can be found at [Git Ready](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html).
+
+### Rebase your pull request ###
+
+If your pull request has a conflict with master, it needs to be rebased:
```
Comment (fnc12) [inline CONTRIBUTING.md]: with `dev`, not `master`

**Comment by soroshsabz** [comment]
@fnc12 thanks for review, I try to resolve these problem soon ;)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
+# How to Contribute #
+
+Thank you for your interest in contributing to the sqlite_orm project!
+
+## GitHub pull requests ##
+
+This is the preferred method of submitting changes.  When you submit a pull request through github,
+it activates the continuous integration (CI) build systems at Appveyor and Travis to build your changesxi
```
[the remark this answers is not in the document]
Comment (soroshsabz) [reply CONTRIBUTING.md]: I change it to changes

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+on a variety of Linux and Windows configurations and run all the test suites.  Follow these requirements
```
[the remark this answers is not in the document]
Comment (soroshsabz) [reply CONTRIBUTING.md]: Yes you right, I add it :)

File: CONTRIBUTING.md (hunk 1 above)
Comment (soroshsabz) [reply CONTRIBUTING.md]: Yes you right, I replace it :)

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+## GitHub pull requests ##
+
+This is the preferred method of submitting changes.  When you submit a pull request through github,
+it activates the continuous integration (CI) build systems at Appveyor and Travis to build your changesxi
+on a variety of Linux and Windows configurations and run all the test suites.  Follow these requirements 
+for a successful pull request:
+
+ 1. All significant changes require a [github issue](https://github.com/fnc12/sqlite_orm/issues).  Trivial changes such as fixing a typo or a compiler warning do not.
+
+ 1. All pull requests should contain a single commit per issue, or we will ask you to squash it.
```
[the remark this answers is not in the document]
Comment (soroshsabz) [reply CONTRIBUTING.md]: @fnc12 I think it is increasing meaning full commit message and dramatically decreasing number of commits, so we can make easier to track meaningful changes and increasing maintainability.

If you disagree with me, I can remove it ;) but I think it is helpful, if you don't like this condition mandatory for contributing, we can make optional and just says as recommendation for contributors, ok?

[the remark this answers is not in the document]
Comment (fnc12) [reply CONTRIBUTING.md]: Ok, you're. I think it is required or this lib will have 50k commits with 100 features only. Let it stay. Thank you

**Comment by soroshsabz** [comment]
@fnc12 Did you think any issue exists?

thanks :)

**Comment by fnc12** [comment]
@soroshsabz sorry for long waiting. I merged it. Thank you

**Comment by soroshsabz** [comment]
😊

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by fnc12** [comment]
@soroshsabz I don't need one

**Comment by soroshsabz** [comment]
I just thanks for your attention and work

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by fnc12** [comment]
Ah ok. You're welcome. You signing with `Get Outlook for Android` confused me

**Pull request merged**